cdap-ui/app/hydrator/controllers/create/create-studio-ctrl.js

```diff
@@ -76,6 +76,9 @@ class HydratorPlusPlusStudioCtrl {
       let config = {};
       config.artifact = artifact;
       HydratorPlusPlusConfigActions.initializeConfigStore(config);
+      if(rConfig.upgrade){
```

Comment:
nit: space after `if`. Aren't this being caught by linter? are you running gulp watch?

Reply:
I have gulp watch running, linter is not complaining about these